### ISSUE TITLE
Update to use discord.js v9.x

### ISSUE THREAD

**Issue by GeekyDeaks** [GeekyDeaks/discord-destinybot#25]
@unisys12 noted that there is a new major release of discord.js (v9.x) which has some breaking API changes.

http://hydrabolt.github.io/discord.js/#!/docs/tag/master/file/general/Updating%20your%20code

Since we are already using the `Promise` style, this should not be too much trouble to integrate.

**Comment by GeekyDeaks** [comment]
After digging around in the source code for a few hours, I have been unable to find anything that allows modification of `roles` for a specific `member`.  Unfortunately the `sar` feature of the bot which uses this is required for the server I am using it on.  I'll put in a change request to the `discord.js` team and might take a look at implementing it myself in the meantime.

**Comment by unisys12** [comment]
So in the meantime, hold off on any changes or new features? I think that would be the safest bet before moving forward. I am going to copy my FFCBOT project over to a new dir and write tests for it using Jasmine _(just so I can get used to TDD and the project itself is really simple)_ . Then rewrite it using V9 of Discord and see what tests fail and update from there. Just to see what happens. **FOR SCIENCE!**

**Comment by GeekyDeaks** [comment]
Yeah - I think that is sensible.  I am nearly done, but it the testing of the bot that will take the time I think.

Definitely recommend TDD.  I tend to be lazy and skip it and then regret it once the project becomes a little larger (just like this bot).

**Comment by GeekyDeaks** [comment]
Hit a blocker with this - see #28 for gory details...
